src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/prototype/ForSimpleInsDel.java

```diff
@@ -0,0 +1,76 @@
+package org.broadinstitute.hellbender.tools.spark.sv.discovery.prototype;
+
+import com.google.cloud.genomics.dataflow.utils.GCSOptions;
+import htsjdk.variant.variantcontext.VariantContext;
+import org.apache.logging.log4j.Logger;
+import org.apache.spark.api.java.JavaPairRDD;
+import org.apache.spark.api.java.JavaRDD;
+import org.apache.spark.broadcast.Broadcast;
+import org.broadinstitute.hellbender.engine.datasources.ReferenceMultiSource;
+import org.broadinstitute.hellbender.tools.spark.sv.StructuralVariationDiscoveryArgumentCollection;
+import org.broadinstitute.hellbender.tools.spark.sv.discovery.AlignedContig;
+import org.broadinstitute.hellbender.tools.spark.sv.discovery.AlignmentInterval;
+import org.broadinstitute.hellbender.tools.spark.sv.discovery.ChimericAlignment;
+import org.broadinstitute.hellbender.tools.spark.sv.discovery.DiscoverVariantsFromContigAlignmentsSAMSpark;
+import org.broadinstitute.hellbender.tools.spark.sv.utils.SVVCFWriter;
+import scala.Tuple2;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+
+final class ForSimpleInsDel implements VariantDetectorFromLongReadAlignments {
+
+    public void inferSvAndWriteVCF(final JavaRDD<AlignedContig> longReads, final String vcfOutputFileName,
+                                   final Broadcast<ReferenceMultiSource> broadcastReference, final String fastaReference,
+                                   final GCSOptions options, final Logger toolLogger){
```

Comment:
It's kind of annoying to have to keep passing around `GCSOptions` and I think the engine team has said these are being phased out eventually. It looks like it's only ever used if accessing a reference from the Google Genomics API, which we never do. It might be worth asking the engine team if we should still be using it, and/or just pass in null when we create the reference object.